src/composition_root.rs

```diff
@@ -69,23 +70,27 @@ fn create_comp_root() -> CompositionRoot<
             SymptomInputsSubmitterImpl<
                 'static,
                 MemoMapperImpl,
-                TcnKeysImpl<PreferencesImpl>,
+                TcnKeysImpl<PreferencesNoopMock>,
                 TcnApiImpl,
             >,
         >,
     >,
     ObservedTcnProcessorImpl<'static, TcnDaoImpl>,
     MemoMapperImpl,
-    TcnKeysImpl<PreferencesImpl>,
+    TcnKeysImpl<PreferencesNoopMock>,
 > {
     let api = &TcnApiImpl {};
-    let preferences = Arc::new(PreferencesImpl {
-        // unwrap: "Errors that are returned from this function are I/O related,
-        // for example if the writing of the new configuration fails or confy encounters
-        // an operating system or environment that it does not support."
-        // The config is critical in this app, so it's ok to crash if not available.
-        config: RwLock::new(confy::load("coepi").unwrap()),
-    });
+
+    // TODO confy seems not to work on Android
+    // let preferences = Arc::new(PreferencesImpl {
+    //     // unwrap: "Errors that are returned from this function are I/O related,
+    //     // for example if the writing of the new configuration fails or confy encounters
+    //     // an operating system or environment that it does not support."
+    //     // The config is critical in this app, so it's ok to crash if not available.
+    //     config: RwLock::new(confy::load("coepi")),
+    // });
+    let preferences = Arc::new(PreferencesNoopMock {});
```

Comment:
Suggest we use conditional compilation: PreferencesNoopMock for target_os == android.
That way we dont break the functionality for iOS.

Reply:
Draft PR! It's still in progress.